pkg/controller/controller_utils.go

```diff
@@ -1127,11 +1127,6 @@ func RemoveTaintOffNode(ctx context.Context, c clientset.Interface, nodeName str
 
 // PatchNodeTaints patches node's taints.
 func PatchNodeTaints(ctx context.Context, c clientset.Interface, nodeName string, oldNode *v1.Node, newNode *v1.Node) error {
```

Comment:
@jprzychodzen now that we've updated this to include RV in the patch request, is there a reason not to include the same update in CCM [here](https://github.com/jprzychodzen/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/node/helpers/taints.go#L91)? At least this way both components will not conflict with other components in the cluster that would try to set taints on the Node.

Reply:
Done